ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiUserManagementService.scala

```diff
@@ -394,29 +394,29 @@ private[apiserver] final class ApiUserManagementService(
       rights: Set[UserRight],
```

Comment:
I haven't caught it before, but the name of this method is misspelled, it should be either `verifyPartiesExistInIdp` or `verifyPartyExistsInIdp`

Reply:
I will address this in the other PR which is changing lots of lines within this file